velox/exec/tests/UnnestTest.cpp

```diff
@@ -285,6 +285,172 @@ TEST_P(UnnestTest, arrayWithOrdinality) {
   assertQuery(params, expectedInDict);
 }
 
+TEST_P(UnnestTest, arrayWithEmptyUnnestValue) {
```

Comment:
@xiaoxmeng : Do we have a SQL query equivalent with LEFT outer join which can generate the same results ? It might be simpler to check test results using it with sql validation in assertQuery. 

Would be simpler for us to understand semantics as well.